<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseTicketDialog</class>
 <widget class="QDialog" name="chooseTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(151, 154, 208);
background-color: rgba(78, 99, 189,220);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(172, 172, 255);
color: rgb(11, 35, 255);
color: rgb(234, 235, 249);</string>
     </property>
     <property name="text">
      <string>TICKETS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>11</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>220</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(245, 245, 245);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>舱位等级</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>价格</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>余票量</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>购买数量</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/new/prefix1/icon1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>NO.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>$150</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="num1">
        <property name="text">
         <string>120</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="spinBox1"/>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="buy1">
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(46, 46, 202);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_15">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/new/prefix1/icon1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>NO.2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>$120</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="num2">
        <property name="text">
         <string>120</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="spinBox2"/>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="buy2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(46, 46, 202);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/new/prefix1/icon1.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>NO.3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>$100</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="num3">
        <property name="text">
         <string>120</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="spinBox3"/>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="buy3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(46, 46, 202);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
